Readme.md

```diff
@@ -36,6 +36,13 @@ import C/Godeps/_workspace/src/D. This makes C's repo
 self-contained and causes 'go get' to build C with the
 right version of all dependencies.
 
+If you don't want to use `-r`, you must make sure that the `Godeps/_workspace`
```

Comment:
s/don't want to/don't us/

`Godeps/_workspace` is an implementation detail. If you are not using `-r` you should wrap your go commands with `godep`: eg. `godep go install -v ./...`

If you fix these issues then I'll merge it.